src/components/home-governance/voting-card/voting-card-active.tsx

```diff
@@ -0,0 +1,214 @@
+import React, {useEffect, useMemo, useRef} from 'react';
+
+import {Pressable, View} from 'react-native';
+
+import {Color, getColor} from '@app/colors';
+import {
+  Icon,
+  ProgressCircle,
+  ProgressCircleInterface,
+  ProgressLine,
+  ProgressLineInterface,
+  Spacer,
+  Text,
+  TextSum,
+  VotingLine,
+  VotingLineInterface,
+} from '@app/components/ui';
+import {createTheme} from '@app/helpers';
+import {I18N} from '@app/i18n';
+import {
+  GovernanceVoting,
+  ProposalRealmType,
+} from '@app/models/governance-voting';
+
+type VotingCardActiveProps = {
+  id: number;
+  onPress?: (hash: number) => void;
+};
+
+const initialVotes = {
+  yes: 1,
+  no: 1,
+  abstain: 1,
+  veto: 1,
+};
+
+const colorVotes = {
+  yes: Color.graphicGreen1,
+  no: Color.textRed1,
+  abstain: Color.graphicSecond4,
+  veto: Color.textYellow1,
+};
+
+export const VotingCardActive = ({id, onPress}: VotingCardActiveProps) => {
+  const item = useMemo(() => {
```

Comment:
use normal data from item of flat list

Reply:
If I use data from flatlist, I will have to manually write the transfer of all the necessary props 2 times

I also noticed a feature of classes that you can't throw the whole object with all the properties